engine/worker/cmd_download.go

```diff
@@ -85,11 +87,13 @@ func downloadCmd(w *currentWorker) func(cmd *cobra.Command, args []string) {
 			}
 		}
 
+		wd, _ := os.Getwd()
```

Comment:
error handling

Reply:
not useful, if the working directory is empty; it will be downloaded in the worker home directory